adr/2023-07-31-fix-promotion-individual-code-redeemer.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Fix promotion individual code redeemer
```

Comment:
Can you add a changelog instead of an ADR. This is kinda just a bugfix :)

Reply:
Of course. I changed it.